src/commands/deployment/onboard.decorator.json

```diff
@@ -22,10 +22,6 @@
       "description": "Name of the resource group to create new storage account when it does not exist",
       "required": true
     },
-    {
-      "arg": "-k, --key-vault-name <key-vault-name>",
-      "description": "Name of the Azure key vault"
-    },
     {
       "arg": "--service-principal-id <service-principal-id>",
```

Comment:
Are `service-principal-id`, `service-principal-password`, `tenant-id` and `subscription-id` still being used elsewhere?

Reply:
in the onboard command these are used to create account service and table. Isn't this the case?

Reply:
right

Reply:
these are not in config file, these are option values for onboard command only

Reply:
they're also in config file: https://github.com/CatalystCode/spk/blob/master/spk-config.yaml#L32

Reply:
i missed these, should I remove them to avoid confusion?

Reply:
I think for now we can leave them since I believe the onboard command takes them either through config file or through the command itself. Although since it's a one time command, it shouldn't rely on the config file. I'll open a separate discussion for that :)

Reply:
ok

Reply:
The service principal related options are used to authenticate with Azure to create storage account and table if they don't exist. Deleting these options make this command not consistent with other commands to pass all required values without depending on config.yaml.